engine/c.cpp

```diff
@@ -412,4 +412,8 @@ KVDKStatus KVDKGetTTL(KVDKEngine* engine, const char* str, size_t str_len,
                       int64_t* ttl_time) {
   return engine->rep->GetTTL(std::string(str, str_len), ttl_time);
 }
+
+KVDKStatus KVDKPersist(KVDKEngine* engine, const char* str, size_t str_len) {
```

Comment:
Will Persist be obscured by PMem Persist? Maybe just call it RemoveExpire or something?

Reply:
removed.